examples/Remarketing/AddFlightsFeed.cs

```diff
@@ -0,0 +1,411 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V2.Errors;
+using Google.Ads.GoogleAds.V2.Resources;
+using Google.Ads.GoogleAds.V2.Services;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using static Google.Ads.GoogleAds.V2.Enums.FeedAttributeTypeEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.FlightPlaceholderFieldEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.PlaceholderTypeEnum.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V2
+{
+    /// <summary>
+    /// This code example adds a flights feed, creates the associated feed mapping, and
```

Comment:
What do you think of adding a reference to the Update example you have for this in case they want to do that in a related piece of code?

Reply:
Done.